src/peerpod-ctrl/Dockerfile

```diff
@@ -1,19 +1,19 @@
 # Build the manager binary
-FROM --platform=$TARGETPLATFORM quay.io/confidential-containers/golang-fedora:1.20.8-38 as builder
+FROM --platform=$TARGETPLATFORM quay.io/confidential-containers/golang-fedora:1.21.9-38 as builder
 ARG TARGETOS
 ARG TARGETARCH
 ARG CGO_ENABLED=1
 ARG GOFLAGS
 
-WORKDIR /workspace/
+WORKDIR /work
 RUN if [ "$CGO_ENABLED" = 1 ] ; then dnf install -y libvirt-devel && dnf clean all; fi
 # Copy the Go Modules manifests
 COPY peerpod-ctrl/go.mod peerpod-ctrl/go.mod
 COPY peerpod-ctrl/go.sum peerpod-ctrl/go.sum
 COPY cloud-providers cloud-providers
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
-WORKDIR /workspace/peerpod-ctrl/
+WORKDIR /work/peerpod-ctrl/
```

Comment:
Any specific reason to change the path here?

Reply:
To fix the "plugin was built with a different version of package XXX" from Peerpod-ctrl log.

The go build env must same for CAA, Peerpod-ctrl, and the cloud provider plugin.